Make whitesapce invalid in token names (#216)
design-tokens/community-group: technical-reports/format/design-token.md

</aside>

> The name of your design token must not include any whitespace. Otherwise your design token is considered as invalid.

Reviewer: ```suggestion
Design tokens are written in kebab-case. Otherwise your design token is considered as invalid.
```

Reviewer 2: I think this should be left as is. Kebab-case is only one way of avoiding whitespace... `snake_case` or `CamelCase` are fine too.
